README.md

```diff
@@ -12,6 +12,43 @@ Follow instructions to [add a token and start the services](https://kernelci.org
 
 > **Note** The `trigger` service was run only once as it's not currently configured to run periodically.
 
+### Setting up LAVA lab
+
+For scheduling jobs, the pipeline needs to be able to submit jobs to a "LAVA lab" type of runtime and receive HTTP(S) callbacks with results over "lava-callback" service.
+Runtime is configured in yaml file following way, for example:
+```
+  lava-collabora: &lava-collabora-staging
+    lab_type: lava
+    url: https://lava.collabora.dev/
+    priority_min: 40
+    priority_max: 60
+    notify:
+      callback:
+        token: kernelci-api-token-staging
+        url: https://staging.kernelci.org:9100
+```
+
+- url is endpoint of LAVA lab API where job will be submitted.
+- notify.callback.url is endpoint of lava-callback service where LAVA will send results.
+- notify.callback.token is token DESCRIPTION used in LAVA job definition. This part is a little bit tricky: https://docs.lavasoftware.org/lava/user-notifications.html#notification-callbacks
+If you specify token name that does not exist in LAVA under user submitting job, callback will return token secret set to description. If following example it will be "kernelci-api-token-staging".
+If you specify token name that matches existing token in LAVA, callback will return token value (secret) from LAVA, which is usually long alphanumeric string.
```

Comment:
@pawiecz i think now it is fixed, match / matches. :) sorry i missed to commit last time